sandbox/src/Cardano.tsx

```diff
@@ -151,6 +150,7 @@ function CardanoSignTransaction({ bb02 }: Props) {
     ['zero-ttl', 'Transaction with TTL=0'],
     ['tokens', 'Transaction sending tokens'],
     ['delegate', 'Delegate staking to a pool'],
+    ['vote-delegation', 'Delegate vote to a dRep'],
```

Comment:
Please add another one that delegates to a keyHash or scriptHash, or add a dropdown to the existing one to switch the mode so all of them can be tested in the sandbox. The hash ones don't work right now (see other comment).